docker/generate_build_matrix.py

```diff
@@ -21,21 +21,21 @@
 
 def _get_pytorch_version(python_version: str):
     if python_version == '3.9':
-        return '1.11.0'
+        return '1.12.0'
     if python_version in '3.8':
-        return '1.10.2'
+        return '1.11.0'
     if python_version == '3.7':
-        return '1.9.1'
+        return '1.10.0'
     raise ValueError(f'Invalid python version: {python_version}')
 
 
 def _get_torchvision_version(pytorch_version: str):
-    if pytorch_version == '1.10.2':
-        return '0.11.3'
     if pytorch_version == '1.11.0':
         return '0.12.0'
-    if pytorch_version == '1.9.1':
-        return '0.10.1'
+    if pytorch_version == '1.12.0':
+        return '0.13.0'
+    if pytorch_version == '1.10.0':
```

Comment:
```suggestion
    if pytorch_version == '1.10.2':
```

Also nit: can you you sort these checks in order? A bit confusing to jump from 1.11 to 1.12 and back to 1.12.